app/api/public.py

```diff
@@ -22,6 +27,19 @@ def get_states():
     )
 
 
+@api.route('/public/states/info.csv', methods=['GET'])
+def get_states_csv():
+    states = State.query.order_by(State.state.asc()).all()
+    columns = [CSVColumn(label="State", model_column="state"),
```

Comment:
I was planning on that, but the issue I ran into is that the columns need to come out in this exact order or it doesn't work, and the set of columns isn't the same between States/US Daily/Current, so it seemed like just too much to try to cram all this CSV stuff into the model for something that's an auxiliary feature to support an output format we don't even like. I'm open to trying to figure out a better way to organize it though. 

My main concern for States Daily is that we don't need to update multiple places when we add a new column. I'm not sure if we'll ever be adding more columns to the CSV or whether we can freeze it as is as a legacy format, but we should find out.

Reply:
Totally, that makes sense. In that case, would you be okay with moving all the CSV output methods into its own API file like `csv.py`? Especially since we may - fingers crossed - phase out the main reason for these, we may as well keep these column lists out of the main API method files.